.gitignore

```diff
@@ -50,3 +50,7 @@ _sandbox
 env
 venv
 .python-version
+
+
+# Pycharm
+.idea/
```

Comment:
Directories like this should be added to your global .gitignore file, not to project-specific .gitignore files.